CONTRIBUTION.md

```diff
@@ -0,0 +1,31 @@
+# Contributing Guidelines
+Thank you for considering contributing to pybot_discord! pybot_discord is a simple Discord bot that responds to commands and messages. It can also tell jokes.
+
+If you would like to contribute to this repository by making the Library better, create an issue with your idea or improvement. If it's approved, create a change and submit a pull request.
+
+## Code of Conduct
+Before getting started, please read and adhere to the code of conduct. We want to maintain a respectful and welcoming community for all participants
+
+**Here are the basic steps to contribute to pybot_discord script**
+
+- Fork the repository
+- Create a new branch for your changes
+- Make your changes and commit them to your branch
+- Push your changes to your fork
+- Create a pull request
+- Gave us a star 
```

Comment:
give*
Nice if you could correct the typo :)

Reply:
sure